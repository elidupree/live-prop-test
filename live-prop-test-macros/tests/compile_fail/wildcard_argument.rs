use live_prop_test_macros::live_prop_test;

#[live_prop_test(test_five)]
fn five(_: i32) -> i32 {
  5
}

fn test_five(_input: &i32) -> impl FnOnce(&i32) {
  move |result| {
    assert_eq!(*result, 5);
  }
}

fn main() {}
