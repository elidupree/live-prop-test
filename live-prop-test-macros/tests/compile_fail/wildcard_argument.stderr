error: live-prop-test only supports function arguments that are bound by-value as an identifier
 --> $DIR/wildcard_argument.rs:4:9
  |
4 | fn five(_: i32) -> i32 {
  |         ^
